orchestrator/orchestrator.go

```diff
@@ -115,8 +115,12 @@ func (o *Orchestrator) Start(ctx context.Context) error {
 				if err := interop.Configure(ctx, chain); err != nil {
 					errs[i] = fmt.Errorf("failed to configure interop for chain %s: %w", chain.Config().Name, err)
 				}
+				wg.Done()
 			}(i)
 		}
+
+		wg.Wait()
+
```

Comment:
small bug where we weren't waiting for the interop applier to finish configuring